Control coverage/Feature/AppService.md

```diff
@@ -2,5 +2,8 @@
-| <b>ControlId:</b><br>Azure_AppService_Config_Disable_Web_Sockets<br><b>DisplayName:</b><br>Web Sockets should be disabled for App Service. <br><b>Description: </b><br> Web Sockets should be disabled for App Service. | <b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web?<br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/webSocketsEnabled | <b>Passed: </b><br>Web sockets for resource  is disabled.<br><b>Failed: </b><br>Web sockets for resource  is enabled. |
\ No newline at end of file
+| <b>ControlId:</b><br>Azure_AppService_DP_Dont_Allow_HTTP_Access<br><b>DisplayName:</b><br>Use HTTPS for app services.<br><b>Description: </b><br> App Service must only be accessible over HTTPS. |<b> ARM API to list all the App Services in a subscription: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Web/sites? <br> 2018-11-01 <br><b>Properties:</b><br> properties/httpsOnly <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/httpsOnly| <b>Passed: </b><br>HTTPS Only setting is eanbled for all slots in App Service.<br><b>Failed: </b><br>HTTPS Only setting is disabled for any slot in App Service. |
+| <b>ControlId:</b><br>Azure_AppService_DP_Use_Secure_TLS_Version<br><b>DisplayName:</b><br>Use Approved TLS Version in App Service.<br><b>Description: </b><br> Use approved version of TLS for the App Service. |<b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web? <br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/minTlsVersion <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> name <br><br> <b> ARM API to get configuration of an App Service slot: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slotName}/config/web? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/minTlsVersion| <b>Passed: </b><br>Current minimum TLS version is set to either equal or greater than the required min TLS version for all the slots in App Service.<br><b>Failed: </b><br>Current minimum TLS version is less than required min TLS version for any of the slot in App Service. |
```

Comment:
Any reason why "ARM API to list all the App Services in a subscription" is not listed for all Controls, except `Azure_AppService_DP_Dont_Allow_HTTP_Access`? The App Service name from this call is used in any subsequent calls for the same App Service.

Reply:
This is not required post our offline conversation.